src/Microsoft.TemplateEngine.Abstractions/ITemplateParameter.cs

```diff
@@ -68,5 +68,10 @@ public interface ITemplateParameter
         /// Gets the default value to be used if the parameter is passed without value for template instantiation.
         /// </summary>
         string? DefaultIfOptionWithoutValue { get; }
+
+        /// <summary>
```

Comment:
do we need to expose it here? I think it might stay internal to `Orchestration`

Reply:
As discused offline - will be refactored as part https://github.com/dotnet/templating/issues/4665